src/components/InfoViewer/InfoViewer.tsx

```diff
@@ -39,7 +39,7 @@ const InfoViewer = ({
             {info && info.length > 0 ? (
                 <div className={b('items')}>
                     {info.map((data, infoIndex) => (
-                        <div className={b('row')} key={data.label + infoIndex}>
+                        <div className={b('row')} key={infoIndex}>
```

Comment:
It's a bad practice to generate keys on the fly: https://react.dev/learn/rendering-lists#rules-of-keys

Reply:
As far as we are not going to manipulate elements in this array, we may still use `infoIndex`, but more proper way is to add ids to `info` (maybe we can use `names` - as I remember, they are unique).